src/main/features/ccj/routes/pay-by-set-date.ts

```diff
@@ -9,9 +9,13 @@ import { DraftService } from 'services/draftService'
 import { ErrorHandling } from 'common/errorHandling'
 import { DraftCCJ } from 'ccj/draft/draftCCJ'
 import { Draft } from '@hmcts/draft-store-client'
+import { MomentFactory } from 'common/momentFactory'
 
 function renderView (form: Form<PayBySetDate>, res: express.Response): void {
-  res.render(Paths.payBySetDatePage.associatedView, { form: form })
+  const futureDate: string = MomentFactory.currentDate().add(30, 'days').format('DD MM YYYY')
```

Comment:
why 30?
why do you format it this way? [if this is going to be displayed we should format it in the view IMO.

Reply:
30 was chosen by marc so that the user always sees a date in the future. 
this format is to give an example for the date fields in our form which require a day, month and year in that format. 
I'll move the formatting to the view

Reply:
done